src/bin/tikv-server.rs

```diff
@@ -650,6 +734,33 @@ fn get_rocksdb_lock_cf_option(config: &toml::Value, total_mem: u64) -> ColumnFam
     cf_opts
 }
 
+fn get_raft_rocksdb_default_cf_option(config: &toml::Value, total_mem: u64) -> ColumnFamilyOptions {
```

Comment:
so we put RegionLocalState, RaftState, RaftEntry in one cf?   use default cf or raft cf?